app/views/pages/home.html.haml

```diff
@@ -8,10 +8,11 @@
 %h4.text-center= t('.why_is_worth')
 %h5= t('.we_have_several_reasons')
 .list-group
-  %a.list-group-item.list-group-item-action{ href: '#' }= t('.less_failures')
-  %a.list-group-item.list-group-item-action{ href: '#' }= t('.increase_value')
-  %a.list-group-item.list-group-item-action{ href: '#' }= t('.makes_you_happy')
-= image_tag 'home/motohelper.jpg', class: 'rounded mx-auto d-block'
+  %button.list-group-item.list-group-item-action{ type: 'button' }= t('.less_failures')
+  %button.list-group-item.list-group-item-action{ type: 'button' }= t('.increase_value')
+  %button.list-group-item.list-group-item-action{ type: 'button' }= t('.makes_you_happy')
```

Comment:
Why do you use buttons? Buttons should attached to an action :) These are only information. The 'hover' behavior is confusing.

Reply:
For me, it's just more dynamically. But I'm not front-end developer or UI/UX guy, so maybe you have right. I'll improve it.